src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null
+                || password.length() == 0
+                || !password.equals(repeatPassword)
+                || password.length() < 10) {
```

Comment:
Можна прибрати умову яка дублюється
```suggestion
        if (password == null
                || !password.equals(repeatPassword)
                || password.length() < 10) {
```